### PR TITLE
Samplerouteupdate

### PR DESCRIPTION
**Pull request by Dezedene**
Display two poi per keyword around the user at loading.

### CODE REVIEW

File: routes/pointofinterest/route.js [hunk 1]
```diff
@@ -1,22 +1,73 @@
...
+// get two closest pois per keyword in a 3 km radius around the user at loading
+router.get('/sample/:latitude/:longitude', (req, res) => {
+  const latToUse = req.params.latitude;
+  const longToUse = req.params.longitude;
+  connection.query(`${getSamplePoisInfos} WHERE keyword.name = 'Sport' ${getSamplePoisInfosCriteria} 
+  UNION (${getSamplePoisInfos} WHERE keyword.name = 'Nature' ${getSamplePoisInfosCriteria})
+  UNION (${getSamplePoisInfos} WHERE keyword.name = 'Attraction' ${getSamplePoisInfosCriteria})
+  UNION (${getSamplePoisInfos} WHERE keyword.name = 'Monument' ${getSamplePoisInfosCriteria})
+  UNION (${getSamplePoisInfos} WHERE keyword.name = 'Utilitaire' ${getSamplePoisInfosCriteria})
+  UNION (${getSamplePoisInfos} WHERE keyword.name = 'Hygiène' ${getSamplePoisInfosCriteria});`,
```
Comment (erabin05) [inline routes/pointofinterest/route.js]: Why the query isn't in a variable like all the other one ?

**Review by erabin05 — CHANGES_REQUESTED** [review]
- A SQL query is not in a variable like the other one

File: routes/pointofinterest/route.js [hunk 2]
```diff
@@ -1,22 +1,73 @@
...
+  [latToUse, longToUse, latToUse,
+    latToUse, longToUse, latToUse,
+    latToUse, longToUse, latToUse,
+    latToUse, longToUse, latToUse,
+    latToUse, longToUse, latToUse,
+    latToUse, longToUse, latToUse], (err, datas) => {
```
Comment (erabin05) [inline routes/pointofinterest/route.js]: the same for this array, for a cleaner code you could create variable

File: routes/pointofinterest/route.js (hunk 1 above)
Comment (Dezedene) [reply routes/pointofinterest/route.js]: The problem has been corrected, SQL query has been set in a variable.

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**